crates/sync/src/commands.rs

```diff
@@ -0,0 +1,4 @@
+#[tauri::command]
+pub async fn alchemy_supported_network(chain_id: u32) -> bool {
```

Comment:
when I read the `app.rs` above I didn't realize this.
shouldn't this be `alchemy_supports_network` (or even `alchemy_is_network_supported` and rename the other function as well)